active/website.md

```diff
@@ -0,0 +1,63 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
```

Comment:
From this description it’s unclear to me if this working group also works on the website’s contents, or just specific things that count as "features"?

For example if someone from our Security team wants to publish a post on the website’s blog – do they reach out to this working group, or elsewhere?

Reply:
I had asked for help on this particular item and you are the one who provided the wording that is on there. I'm happy if you can add on that you feel would make sense.

Reply:
@ronnzw cool! Let‘s see what others think then.

Reply:
[According this thread](https://github.com/django/dsf-working-groups/pull/10/files#r1469462925) and if no one disagree, we keep the fact that the content is not written by us, it should be only specific things that count as "features" or things.

Reply:
Cool!